src/Controller/StatusController.php

```diff
@@ -61,8 +63,23 @@ public function statusAction(Request $request)
     private function verifyNoDatabaseCacheIssues(): bool
     {
         try {
-            $pid = new Pid('b006m86d'); //Eastenders
-            $this->get('pps.programmes_service')->findByPidFull($pid);
+            // Eastenders clip
+            $clipPid = new Pid('p04r0jcv');
+            $this->get('pps.programmes_service')->findByPidFull($clipPid);
+
+            // Broadcast
+            $fromDateTime = new DateTimeImmutable('2010-01-15 06:00:00');
```

Comment:
Any reason for these specific dates?

Reply:
Nothing in particulat, they are the same date as in the testFindByServiceAndDateRange unitest